openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityThreadPanel/ActivityThreadPanel.tsx

```diff
@@ -220,25 +217,29 @@ const ActivityThreadPanel: FC<ActivityThreadPanelProp> = ({
   const [showNewConversation, setShowNewConversation] =
     useState<boolean>(false);
 
-  const [confirmationState, setConfirmationState] = useState<ConfirmState>({
-    state: false,
-    threadId: undefined,
-    postId: undefined,
-  });
+  const [confirmationState, setConfirmationState] = useState<ConfirmState>(
+    confirmStateInitialValue
+  );
 
-  const onDiscard = () => {
-    setConfirmationState({
-      state: false,
-      threadId: undefined,
-      postId: undefined,
+  const getThreads = () => {
+    getAllFeeds(threadLink).then((res: AxiosResponse) => {
```

Comment:
We should handle errors in a catch block

Reply:
Hey @vivekratnavel, sure will take care of this in the next patch.